### ISSUE TITLE
Percona application pod goes to 'CrashLoopBackOff' on failing one replica out of 3 replica's.

### ISSUE THREAD

**Issue by dargasudarshan** [openebs/openebs#1563]
## Is this a BUG REPORT or FEATURE REQUEST?
BUG Report
 
 Choose one: BUG REPORT or FEATURE REQUEST

**What happened**:
1. I have k8s setup with 1 master and 3 minions (Version 1.8.8)
2. Deployed percona application with Liveness probe check.
3. Percona application created with 3 replicas and running.
4. Then delete one of the replica to check the quorum condition reliability i.e >51% of replica's are available.
>> Here observed that Percona application pod goes to 'CrashLoopBackOff'
```
ubuntu@kubemaster-01:~$ kubectl get pods -n replica-quorum
NAME                                                             READY     STATUS             RESTARTS   AGE
percona-dcf7878c7-b6hkn                                          0/1       CrashLoopBackOff   6          13m
pvc-91f2a709-63e4-11e8-b760-02b983f0a4db-ctrl-74d5b7b857-wlfzb   2/2       Running            0          13m
pvc-91f2a709-63e4-11e8-b760-02b983f0a4db-rep-7b66c7f48d-ldbpc    1/1       Running            0          13m
pvc-91f2a709-63e4-11e8-b760-02b983f0a4db-rep-7b66c7f48d-rh6nd    1/1       Running            0          8m
```
Logs shows that the mount point is in Read Only state.
```
ubuntu@kubemaster-01:~$ kubectl logs -f percona-dcf7878c7-b6hkn -n replica-quorum
chown: changing ownership of '/var/lib/mysql/ibdata1': Read-only file system
chown: changing ownership of '/var/lib/mysql/lost+found': Read-only file system
chown: changing ownership of '/var/lib/mysql/ib_logfile101': Read-only file system
chown: changing ownership of '/var/lib/mysql/': Read-only file system
```
**What you expected to happen**:
Application should not crash when there are at least 2 replica's up and running.

**How to reproduce it (as minimally and precisely as possible)**:
1. I have k8s setup with 1 master and 3 minions (Version 1.8.8)
2. Deployed percona application with Liveness probe check.
3. Percona application created with 3 replicas and running.
4. Then delete one of the replica to check the quorum condition reliability i.e >51% of replica's are available.

**Anything else we need to know?**:
```Version openebs/jiva:ci```

**Comment by dargasudarshan** [comment]
[replica_quorum_failure_test.log](https://github.com/openebs/openebs/files/2053683/replica_quorum_failure_test.log)
Attaching the stern logs.

**Comment by JohnnyQQQQ** [comment]
Seems to be the same as #1557

**Comment by dargasudarshan** [comment]
This issue is fixed in 0.7.0
